keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -1100,13 +1107,46 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason: `Bucket provisioning 'bucket-name' failed`
-- [ ] API .status
-  - Condition `PodReady=False "Error: secrets 'bucket-secret' not found"`) 
-  - Other field:
-- [ ] Other (treat as last resort)
-  - Details:
+WIP: investigating status items
+Using external snapshotter as ref: https://github.com/kubernetes-csi/external-snapshotter/blob/533a2ee13a39cc4e568fb9181bdac4c2a7d03bd6/client/apis/volumesnapshot/v1/types.go#L131-L191
+
+- Bucket
+  - [ ] Events
+    - FailedCreateBucket - Report when COSI fails to create a bucket, with error message
+	  - FailedDeleteBucket - Report when COSI fails to delete a bucket, with error message
+  - [ ] API .status
+    - [x] BucketReady bool
+    - [ ] ErrorMessage string - last error message; cleared when provisioning is successful
```

Comment:
I think we might be able to do more than ErrorMessage here. Check out the volume snapshot KEP's error. It's a struct that includes both the time of the error as well as the error message:
https://github.com/kubernetes-csi/external-snapshotter/blob/533a2ee13a39cc4e568fb9181bdac4c2a7d03bd6/client/apis/volumesnapshot/v1/types.go#L178-L185